packages/devtools_app/lib/main.dart

```diff
@@ -9,50 +9,10 @@ import 'src/analytics/stub_provider.dart'
 import 'src/app.dart';
 import 'src/config_specific/framework_initialize/framework_initialize.dart';
 import 'src/config_specific/ide_theme/ide_theme.dart';
-import 'src/config_specific/load_fallback_app/load_fallback_app.dart';
 import 'src/debugger/syntax_highlighter.dart';
 import 'src/preferences.dart';
 
 void main() async {
-  final defaultOnError = FlutterError.onError;
```

Comment:
Can you please also submit a CL with this change to the internal main.dart file?

Reply:
good news is as the files it depends on were deleted you should get a compile error as part of the next roll. I can do the next roll into internal and fix as part of that.